CMakeLists.txt

```diff
@@ -176,8 +176,15 @@ endif()
 
 find_program(clang_tidy_path NAMES clang-tidy)
 if (clang_tidy_path)
+    set(pgp-packet-includes
```

Comment:
Isn't it nicer to use CMake's [file glob](https://cmake.org/cmake/help/latest/command/file.html) for this?

Reply:
We discussed it with @omartijn, and the problem is that if someone adds a new header file without rerunning `cmake`, the header will not be included in the check. So that's why I went with this.